cmd/fdsn-holdings-consumer/main.go

```diff
@@ -80,38 +124,44 @@ func main() {
 	db.SetMaxIdleConns(p.MaxIdle)
 	db.SetMaxOpenConns(p.MaxOpen)
 
+	// provide a soh heartbeat
+	health := health.New(healthCheckService, healthCheckAged, healthCheckStartup)
+
 ping:
 	for {
 		err = db.Ping()
 		if err != nil {
 			log.Println("problem pinging DB sleeping and retrying")
+			health.Ok() //send heartbeat
```

Comment:
Not necessary. If the service stuck during initial DB connection then it's unhealthy